test/check-application

```diff
@@ -2259,12 +2269,42 @@ class TestFiles(testlib.MachineCase):
                     b.click("button.pf-m-secondary:contains('Keep original')")
 
             b.wait_not_present(".pf-v5-c-modal-box")
-            b.wait_in_text(".pf-v5-c-alert__description", "Successfully uploaded file")
-            b.click(".pf-v5-c-alert__action button")
-            b.wait_not_present(".pf-v5-c-alert__action")
+            assert_upload_alert([os.path.basename(files[-1])], "admin:admin")
 
             self.assertEqual(m.execute(f"cat {dest_dir}/{filename}"), "new content")
 
+            # As administrator upload in testuser and change ownership to root:root
+
+            m.execute("useradd --create-home testuser")
+            b.go('/files#/?path=/home/testuser')
+            b.wait_not_present('.pf-v5-c-empty-state')
+
+            filename = os.path.basename(files[-1])
+            b.click("#upload-file-btn")
+            b.upload_files("#upload-file-btn + input[type='file']", [files[-1]])
+            assert_upload_alert([filename], "testuser:testuser", close=False)
+            self.assert_owner(f'/home/testuser/{filename}', 'testuser:testuser')
+
+            b.click(".pf-v5-c-alert__action-group button")
+            b.wait_not_present(".pf-v5-c-alert__action")
+            b.wait_in_text(".pf-v5-c-modal-box__title-text", filename)
+            # BUG BUG BUG gets all values but not the specific selected value
```

Comment:
That should now be fine, but other tests have `wait_in_text` so I'll handle that in a separate PR

Reply:
Wait, what? `wait_val()` *is* supposed to give you the specific current value, not a list of all of them. Is this comment obsolete, or do we have a bug in our select or testlib?